### PR TITLE
Remove support for influxdb akka metrics

### PR DESCRIPTION
**Pull request by simao**
This means finagle metrics are no longer used as they only support
influxdb.

We don't support influxdb on our platform anymore

Add prometheus metrics listener

Due to the way prometheus aggregates counters, we should have the queue
name on the metric name. This currently requires a query in prometheus
like:

`bus-listener.+`

But these metrics all have the same labels, so we cannot use the prometheus
`by()` operator, so we cannot properly sum these metrics in grafana.

This also means we cannot calculate rates on these metrics, because:

`rate({__name__=~"bus_listener.+_processed"}[5m])`

fails with `vector cannot contain metrics with the same labelset`, because
different metric names only contain the `__name__` label.

DropWizard doesn't allow us to set labels on metrics, so we need to use the
prometheus client directly. This way the name of the queue goes in a label
and the metrics have the same name and can be aggregated by queue name.

### CODE REVIEW

File: libats-metrics-finagle/src/main/scala/com/advancedtelematic/metrics/FinagleMetricsSink.scala [hunk 1]
```diff
@@ -3,19 +3,8 @@
  */
```
Comment (jochenschneider) [inline libats-metrics-finagle/src/main/scala/com/advancedtelematic/metrics/FinagleMetricsSink.scala]: Why is this file still needed?

Comment (simao) [reply libats-metrics-finagle/src/main/scala/com/advancedtelematic/metrics/FinagleMetricsSink.scala]: No, deleted.
